src/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -142,4 +142,11 @@
     <protected>false</protected>
     <shortDescription>Abandon Request</shortDescription>
   </labels>
+  <labels>
+    <fullName>cookbook_MainTitle</fullName>
+    <value>Hi, what do you need help with?</value>
```

Comment:
@acarpentier01  please tell me if this label and the french version of it looks fine to you.

Reply:
@mmitiche The current labels are fine but not a word-for-word match. Here are a couple of options:
1. Hi, how may we help you? Bonjour, comment pouvons-nous vous aider?
2. Hi, need help? Bonjour, besoin d'assistance?

Reply:
Thank you 👌